### PR TITLE
Document onok

### PR DESCRIPTION
**Pull request by johlrogge**
Changes `Fn` to `FnMut` and documents on_ok

### CODE REVIEW

File: src/onok.rs [hunk 1]
```diff
@@ -8,21 +8,36 @@
...
-    F: Fn(&O),
+    F: FnMut(&O),
 {
+    /// Perform a side effect on each Ok value
+    ///
+    /// ```
+    /// use resiter::onok::OnOkDo;
+    /// use std::str::FromStr;
+    /// let mut oks = Vec::new();
+    ///
```
Comment (matthiasbeyer) [inline src/onok.rs]:
```suggestion
    ///
    /// let mut oks = Vec::new();
```

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Only this small nitpick. It's a bit more readable IMO.

**Comment by matthiasbeyer** [comment]
Feel free to fixup and squash and then merge yourself!

bors delegate=johlrogge

**Comment by johlrogge** [comment]
bors merge

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
bors retry

**Pull request merged**